mantidproject/mantid
# Fix release note warning preventing Windows Nightly build

**Description of work.**
A release note was added to a folder entitled 'Improvements' which has caused an unstable Windows Build and is preventing the building of the latest Windows Nightly package. Release notes should be added to either a 'Bugfixes' or a 'New_features' folder only (see https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html#file-location). It also meant the release note was not visible in the release notes.

This PR moves the note which should fix the unstable build problem and make the release note visible in the user facing release notes.

**To test:**

- Check that release note 33708 is now in docs/source/release/v6.4.0/Workbench/New_features
- Do a clean build of the documents and check the 33708 release note now appears on the Workbench page.
- Check that there are no release notes in folders named 'Improvement'.

*There is no associated issue.*

*This does not require release notes* because **it is a fix to unstable builds**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
